thousandeyessdk/test.py

```diff
@@ -30,5 +30,24 @@ def domain(self):
     def country_id(self):
         return self._data.get('countryId', None)
 
+    def get_detailed_path_trace(self, agent_id, round_id):
```

Comment:
agent_test should be removed when this was added

Reply:
Yup, only after the other projects migrate to this ver. (`0.0.6a8`) and stop using `AgentTest`. Otherwise there's a chance I'll block other devs work.